openreview/venue/matching.py

```diff
@@ -1155,6 +1155,8 @@ def deploy_sac_assignments(self, assignment_title, overwrite):
     def deploy(self, assignment_title, overwrite=False, enable_reviewer_reassignment=False):
 
         self.venue.invitation_builder.set_assignment_invitation(self.match_group.id)
+        recruitment_invitation_id=self.venue.get_invitation_id('Proposed_Assignment_Recruitment', prefix=self.match_group.id)
+        self.venue.invitation_builder.expire_invitation(recruitment_invitation_id)
```

Comment:
do you mind to move this line here?

```
self.venue.invitation_builder.expire_invitation(self.venue.get_assignment_id(self.match_group.id))
```

in case that someone is editing the proposed assignments while we are deploying the edges